enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -12,13 +13,33 @@ approvers:
   - "@ewolinetz"
   - "@richm"
 creation-date: 2019-09-17
-last-updated: 2019-10-25
+last-updated: 2020-02-21
 status: implementable
 see-also:[]
 replaces:[]
 superseded-by:[]
 ---
 
+<!-- 
+
+XXX(alanconway) comments here and in the code indicate points that need review.
+
+NOTE: Changed name from LogForwarding to 'Forwarder'. CR names should be
+nouns, the CR is defined in the 'logging' package so the 'log' prefix is
```

Comment:
While I agree it is redundant I lean in the direction that user's rarely retrieve or interact objects by including the namespace (`logging.openshift.io`)  which is what I think you are saying here.  It's clearer to me to `oc get logforwarder` then it is `oc get forwarder` which is what this will translate into.

Reply:
+1

Reply:
Changed to ClusterLogForwarder